oauth/index.php

```diff
@@ -1,72 +1,91 @@
 <?php
 session_start();
-?>
+/**
+ * @author Denis CLAVIER <clavierd at gmail dot com>
+ * A modified verion by dimst23
+ */
 
-<!DOCTYPE html>
-<html>
-	<head>
-	    <link rel="stylesheet" type="text/css" href="./style.css">
-		<title>LDAP Connection Interface</title>
-	</head>
 
-	<body>
-		<center>
-			<table background="images/login.png" border="0" width="733" height="348" cellspacing="1" cellpadding="4">
-				<tr>
-					<td width="40%">&nbsp;</td>
-					
-					<td width="60%">
-						<table border="0" width="100%">
+// include our LDAP object
+require_once __DIR__.'/LDAP/LDAP.php';
+require_once __DIR__.'/LDAP/config_ldap.php';
 
-							<tr>
-								<td align="center">
-									<div class="LoginTitle">LDAP Authentification</div>
-						
+$prompt_template = new DOMDocument();
```

Comment:
It should be also updated in the README.md that this line requires the apt package "php-dom" to be installed.

> apt-get -y install php-dom